general/nuxt.config.js

```diff
@@ -187,18 +187,8 @@ export default {
   },
 
   proxy: [
-    process.env.VUE_RESOURCES_BASE_URL.startsWith("http://") ||
-    process.env.VUE_RESOURCES_BASE_URL.startsWith("https://")
-      ? process.env.VUE_RESOURCES_BASE_URL
-      : process.env.VUE_BASE_URL +
-        process.env.VUE_ONTOLOGY_NAME +
-        "/ontology/api",
-    process.env.VUE_RESOURCES_BASE_URL.startsWith("http://") ||
-    process.env.VUE_RESOURCES_BASE_URL.startsWith("https://")
-      ? process.env.VUE_RESOURCES_BASE_URL
-      : process.env.VUE_BASE_URL +
-        process.env.VUE_ONTOLOGY_NAME +
-        "/ontology/*/api",
+    "https://spec.edmcouncil.org/" + process.env.VUE_ONTOLOGY_NAME + "/ontology/api",
+    "https://spec.edmcouncil.org/" + process.env.VUE_ONTOLOGY_NAME + "/ontology/*/api"
```

Comment:
If the conditions that were there did not cause problems then I would like to see them restored. If they were causing problems then they need to be reworked/add a solution to them.